README.md

```diff
@@ -1,50 +1,34 @@
 # BigriverBookstore-api
 
-The jsonapi for the bigriver bookstore
+## Installation
```

Comment:
Please leave this sections as they are and update the sections marked with TBD (to be discussed) bellow.